en/ui-system/components/engine/ui-material.md

```diff
@@ -0,0 +1,47 @@
+# UI Custom Material
+We supports UI custom material in sprite component since v1.2, the user interface is as follows:
+
+![](ui-material/UIMaterial.png)
+
+Its usage is no different from other materials, but because the SpriteComponent has some function based on the UI-built-in materials, there are some points to note:
```

Comment:
Its usage is no different from other materials, but because the SpriteComponent has some function based on the UI-built-in materials, there are some points to note:
-->
Using a custom material works the same as the UI_built-in materials. However, there are few items to take into consideration: